web/js/components/location-search/coordinates-dialog.js

```diff
@@ -83,7 +85,7 @@ class CoordinatesDialog extends Component {
       <>
         <span
           id={closeButtonId}
-          className={`close-tooltip ${closeButtonId}`}
+          className={`close-tooltip ${closeButtonClassName}`}
           onTouchEnd={this.clearCoordinates}
```

Comment:
Place markers can't be removed on mobile and I believe this is why:

It looks like a different function is called `onTouchEnd` for this `span` as compared to the `onClick` for the `<FontAwesomeIcon>` element contained within.  `this.clearCoordinates` is not even defined in this file anymore.  And I think we want to remove the `CLEAR_MARKERS` action/constant/reducer as well as where it's being picked up in `map/ui.js` and causes `removeAllCoordinatesMarkers()` to be called.